src/Audio_Registers.md

```diff
@@ -0,0 +1,383 @@
+
+This register controls CH3's DAC.
+Like other channels, turning the DAC off immediately turns the channel off as well.
+
+```
+Bit 7 - Sound Channel 3 DAC  (0=Off, 1=On)
+```
+
+The channel is often turned off just before writing to [wave RAM](<#FF30–FF3F — Wave pattern RAM>) to avoid issues with accessing it; see there for more info.
+
+### FF1B — NR31: Channel 3 length timer \[write-only\]
+
+This register controls the channel's [length timer](<#Length timer>).
+
+```
+Bit 5-0 - length timer
+```
+
+The higher the [length timer](<#Length timer>), the shorter the time before the channel is cut.
+
+### FF1C — NR32: Channel 3 output level
+
+This channel lacks the envelope functionality that the other three channels have.
+
+```
+Bits 6-5 - Output level selection
+```
+
+Bits 6-5 (binary) | Output level
+-----------------:|--------------
+  %00             | Mute (No sound)
+  %01             | 100% volume (use samples read from Wave RAM as-is)
+  %10             |  50% volume (shift samples read from Wave RAM right once)
+  %11             |  25% volume (shift samples read from Wave RAM right twice)
+
+### FF1D — NR33: Channel 3 wavelength low \[write-only\]
+
+This register stores the low 8 bits of the channel's 11-bit "[wavelength](<#Frequency>)".
+The upper 3 bits are stored in the low 3 bits of `NR34`.
+
+The actual signal frequency is <math><mfrac><mn>65536</mn><mrow><mn>2048</mn><mo>-</mo><mi>wavelen</mi></mrow></mfrac></math> Hz: the higher the value, the higher the frequency.
+This is the whole wave's frequency; the rate at which the channel steps through the 8 "indices" in its wave form is 32 times that, i.e. <math><mfrac><mn>2097152</mn><mrow><mn>2048</mn><mo>-</mo><mi>wavelen</mi></mrow></mfrac></math>) Hz = <math><mfrac><mn>2</mn><mrow><mn>2048</mn><mo>-</mo><mi>wavelen</mi></mrow></mfrac></math> MiHz.
+
+::: warning DELAY
+
+Modifications of the frequency (via `NR33` or `NR34`) only take effect after the following time wave RAM is read.
+([Source](https://github.com/LIJI32/SameSuite/blob/master/apu/channel_3/channel_3_freq_change_delay.asm))
+
+:::
+
+### FF1E — NR34: Channel 3 wavelength high & control
+
+```
+Bit 7   - Trigger (1=Restart Sound)    (Write Only)
+Bit 6   - Sound Length enable          (Read/Write)
+          (1=Stop output when length in NR31 expires)
+Bit 2-0 - "Wavelength"'s higher 3 bits (Write Only)
+```
+
+Writing a value here with bit 7 set [triggers](<#Triggering>) the channel.
+
+::: warning RETRIGGERING CAUTION
+
+On monochrome consoles only, retriggering CH3 while it's about to read a byte from wave RAM causes wave RAM to be corrupted in a generally unpredictable manner.
+
+:::
+
+Bit 6 takes effect immediately upon writing to this register.
+
+::: warning PLAYBACK DELAY
+
+Triggering the wave channel does not immediately start playing wave RAM; instead, the *last* sample ever read (which is reset to 0 when the APU is off) is output until the channel next reads a sample.
```

Comment:
```suggestion
Triggering the wave channel does not immediately start playing wave RAM; instead, the *last* sample ever read (which is reset to 0 when the APU is off) is played until the channel next reads a sample.
```

Reply:
"or 0 if none was since the APU was last turned on" sounds still very confusing to read ..

Reply:
Edited, I had a word.